README.md

```diff
@@ -16,6 +16,8 @@ The basic host test only monitors the device's default serial port (the serial c
 +------------------+                               +-------------------------+
 ```
 
+Test runner on the target writes the test case properties like timeout, host test name, description and test_id on the serial port. This helps host test runner to find the required python module in the host tests store for executing the host side of the test.
```

Comment:
Where are this properties described? test_id for instance? I could not locate any docs in this repo